databases/week3/week3hw.sql

```diff
@@ -0,0 +1,166 @@
+USE meal_sharing;
+
+-- ********************** Meal **********************
+-- Get all meals
+SELECT*
+FROM meal;
+
+-- Add a new meal
+INSERT INTO meal
+VALUES
+    ( 1,
```

Comment:
`meal_id` is `auto_increment` so you MUST NOT add a custom number. It could cause problems with the internal alto incremental algorithm. 
So, if you have an `auto_increment` field - don't provide a value for it